ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -322,6 +360,8 @@ open class ShimmerView: UIView {
 
 public extension Colors {
 	struct Shimmer {
+		public static var darkGradient = UIColor.black
```

Comment:
super nit: ithink it darkGradient: UIColor = .black  would be more consistent with other files